src/components/Toolbar/_mixins.scss

```diff
@@ -199,13 +199,12 @@ $toolbar__panel__transition__timing-function: $timing-function;
   }
 
   &__nav__item__icon {
-    width: carbon--mini-units($count: 2);
-    height: carbon--mini-units($count: 2);
+    width: carbon--mini-units($count: 2.5);
+    height: carbon--mini-units($count: 2.5);
+    margin-left: -35px;
```

Comment:
whats the negative margin for?

Reply:
I need to use a negative margin to draw the element closer to the left neighbour to align it with items without icons